EIPS/eip-7886.md

```diff
@@ -0,0 +1,400 @@
+author: Francesco D`Amato (@fradamt), Toni Wahrstätter (@nerolation)
+discussions-to: https://ethereum-magicians.org/t/eip-xxxx-delayed-execution/22890
+status: Draft
+type: Standards Track
+category: Core
+created: 2025-02-18
+---
+
+
+## Abstract
+
+This proposal makes (execution) blocks statically verifiable through minimal checks that only require the previous state, but no execution of the block's transactions, allowing validators attest to a block's validity without completing its execution. We allow transactions to be skipped when invalid at execution time, without invalidating the whole block. To ensure that even skipped transactions pay for their resources, the `COINBASE` pays for all inclusion costs upfront (base cost, calldata and blobs), and recovers the costs only when transactions are successfully executed.
+
+## Motivation
+
+A key advantage of the skipping mechanism is that it enables **asynchronous block validation**: a block can be considered valid even if it contains invalid transactions, because those transactions end up skipping execution without invalidating the entire block. 
+
+## Overview
+
+Enabling delayed execution by making the block's validity statically verifiable requires two things:
+
+1. **Deferred execution outputs**: all header fields that commit to execution outputs are deferred by one slot. For example `state_root` and `receipt_root` become `pre_state_root` (similarly to [EIP-7862](./eip-7862.md)) and `parent_receipt_root`, the root of the state and receipt trie obtained after executing the block's parent. The same applies to the General Purpose Execution Layer Requests from [EIP-7685](./eip-7685.md): the requests are deferred by one slot and the requests in the CL must correspond to the `parent_requests_hash` in the EL header. However, this alone would only defer the computation of these execution outputs (mainly of the state root) rather than the actual execution, because verifying transaction validity would still require executing. 
+2. **Upfront payment of inclusion costs by `COINBASE`**: in addition, we need to be able to skip (no-op) invalid transactions without invalidating the whole block. Right now, this is not possible because of the free-DA problem: as soon as we include a transaction into a block, it must pay for its data footprint. By charging the inclusion cost of all transactions upfront from the block's `COINBASE`, it is possible to skip transactions that are found to be invalid at execution time, because the protocol has already been compensated for the inclusion.
+
+## Specification
+
+### Change to the header structure
+
+#### Deferred fields 
+
+In order to be verifiable before execution, the header cannot commit to any execution output. In particular, we need to defer these fields: `state_root, receipt_root, bloom, gas_used, requests_hash`. We replace them with the equivalent execution outputs from the parent block. 
```

Comment:
> In order to be verifiable before execution, the header cannot commit to any execution output.

A bit of rationale in the specification section is fine (this is at the limit) but try to keep the Specification section purely about the changes that need to be made and not about the reasoning behind the changes.